src/components/UI/docs/MDComponents.tsx

```diff
@@ -15,7 +15,7 @@ import { Code, Note } from '.';
 import { textStyles } from '../../../theme/foundations';
 import { parseHeadingId } from '../../../utils/parseHeadingId';
 
-const { header1, header2, header3, header4 } = textStyles;
+const { h1, header2, header3, header4 } = textStyles;
```

Comment:
why do we have `header2`, `header3`, etc instead of `h2`, `h3`, etc?

Reply:
Will see what I can bring over to simplify